cli/src/torii/routing.rs

```diff
@@ -232,15 +202,18 @@ async fn handle_pending_transactions(
     sumeragi: SumeragiHandle,
     pagination: Pagination,
 ) -> Result<Scale<Vec<VersionedSignedTransaction>>> {
+    // TODO: Don't clone wsv here
```

Comment:
Is it applicable here? It's transactions, so I don't think so

Reply:
it's pending transactions. Those are in the queue. Look at how `wsv` is used after this close, only by reference

Reply:
I see